docs/pages/components/notification.md

```diff
@@ -8,8 +8,7 @@ permalink: components/notification.html
 folder: components
 summary:
 ---
-
-{: .docs-intro}
+{: .docs-intro .docs-intro--def-margin}
```

Comment:
this should not be part of this PR

Reply:
The PR changes all the margin-left's in the documentation. There is nothing touching the library so it should be all part of same pr as it is all in the same scope
@droshev